src/main/java/com/iemr/tm/service/schedule/SchedulingServiceImpl.java

```diff
@@ -89,7 +89,6 @@ private static String updateString(String slotdetail, Timestamp startTime, Times
 		Integer startslot = ((startTime.getHours() * 60) + (startTime.getMinutes())) / 5;
 		Integer endslot = ((endTime.getHours() * 60) + (endTime.getMinutes())) / 5;
 		// Integer startslot=
```

Comment:
@sachitsinha1 Shouldn’t you remove the comment **// Integer startslot =** as well?

Reply:
yes, doing in parts.